pkg/activator/net/revision_backends_test.go

```diff
@@ -632,9 +632,9 @@ func TestRevisionBackendManagerAddEndpoint(t *testing.T) {
 			if got, want := revDests, tc.expectDests; !cmp.Equal(got, want) {
 				t.Errorf("RevisionDests = %v, want: %v, diff(-want,+got):%s\n", got, want, cmp.Diff(want, got))
 			}
-			time.Sleep(10 * time.Millisecond)
 		})
 	}
+	time.Sleep(informerRestPeriod)
```

Comment:
I feel like this should happen *after* `cancel()`, which is done by defer.

Reply:
Let's try :) It could be better.